src/llm/llm_executor.hpp

```diff
@@ -87,6 +87,7 @@ class LLMExecutorWrapper {
                     stepCounter++;
                     llmExecutor->step();
                 } else {
+                    SPDLOG_LOGGER_INFO(llm_executor_logger, "All requests: {}; Scheduled requests: {}; Cache usage {:.1f}%;", 0, 0, 0.0);
```

Comment:
Can we use `llmExecutor->printMetrics()` call here?

Reply:
no because llmExecutor metrics are refreshed right before the step execution so there will be never empty queue and cache reported.

Reply:
We are printing hardcoded values here anyway so this is not reflecting actual state of cb lib.